src/Icon/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import * as React from 'react';
+import { Text as RNText } from 'react-native';
+import StyleSheet from '../PlatformStyleSheet';
+
+import iconsMap from './icons.json';
+
+export type Props = {|
+  +name: string,
+  +size?: number,
+  +color?: string,
+|};
+
+const getIconCharacter = name => {
+  const icon = iconsMap[name];
+  if (!icon) {
+    throw Error(`Icon with name "${name}" does not exist.`);
+  }
+
+  if (/^E(.{3})$/.test(iconsMap[name].character)) {
+    return String.fromCharCode(parseInt(iconsMap[name].character, 16));
+  }
+  return iconsMap[name].character;
+};
+
+const Icon = ({ name, color, size }: Props) => (
+  <RNText
+    color={color}
+    size={size}
+    style={[styles.icon, { color: color || '#46515e', fontSize: size || 20 }]}
+  >
```

Comment:
`{ color: color ?? '#46515e'`
`fontSize: size ?? 20` -> since passing fontSize 0 would fallback to 20, which is not what you expect